src/System.Xml.XPath.XDocument/ref/System.Xml.XPath.XDocument.Forwards.cs

```diff
@@ -1,7 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Runtime.CompilerServices;
-
-[assembly:TypeForwardedTo(typeof(System.Xml.XPath.Extensions))]
```

Comment:
This wasn't being used and we don't need to typeforward `System.Xml.XPath.Extensions` as it is not exposed in any other ref assembly.